lib/trino-hdfs/src/main/java/io/trino/hdfs/TrinoFileSystemCache.java

```diff
@@ -337,6 +338,7 @@ public String toString()
 
     private static class FileSystemWrapper
             extends FilterFileSystem
+            implements FileSystemWithBatchDelete
```

Comment:
Why is it needed here?
in `HdfsFileSystem` you unwrap all the filtered FS to get the "raw" one, so this should be equivalent.
or maybe that one is not needed?

please help me understand

Reply:
I don't remember now :/ I will delete it and we will see from the CI :D

Reply:
hmm it works which is great :D